src/components/common/ActionSheet/index.tsx

```diff
@@ -1,19 +1,34 @@
 import i18next from 'i18next';
-import React from 'react';
+import React, { RefObject } from 'react';
 import { Text, View } from 'react-native';
 import { Modalize } from 'react-native-modalize';
 import { Portal } from 'react-native-portalize';
 
 import Touchable from '../Touchable';
 import styles from './styles';
 
-function ActionSheet({ modalRef, onClose, title, subtitle, options }) {
+interface Option {
+  id: number;
+  label: string;
+  onPress: () => void;
+  destructive?: boolean;
+}
+
+interface Props {
+  modalRef: RefObject<Modalize>;
+  onClose: () => void;
+  title?: string;
+  subtitle?: string;
+  options: Option[];
+}
+
+function ActionSheet({ modalRef, onClose, title, subtitle, options }: Props) {
```

Comment:
Loving this <3 ![image](https://user-images.githubusercontent.com/44204622/170130899-a27dfdf9-6af4-49ad-adcb-6c5d2c447506.png)

Reply:
😆  we were missing you TS